_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -15907,10 +15907,7 @@
 "aIe" = (
 /obj/machinery/atmospherics/pipe/simple/supply/hidden/layer4,
 /obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer2,
-/turf/open/floor/plating{
-	initial_gas_mix = "o2=0.01;n2=0.01";
```

Comment:
hahaha

Reply:
don't you love variable edits, god knows I do

Reply:
You have no god damn idea how annoying this was, thank you so much